resources/definitions/fdmprinter.def.json

```diff
@@ -1095,10 +1095,8 @@
                     "description": "The smallest line width, as a factor of the normal line width, below which it will choose to use fewer, but wider lines to fill the available space the wall needs to occupy. Reduce this setting to use more, thinner lines. Increase to use fewer, wider lines. Note that this applies -as if- the entire shape should be filled with wall, so the middle here refers to the middle of the object between two outer edges of the shape, even if there actually is fill or (other) skin in the print instead of wall.",
                     "type": "float",
                     "unit": "%",
-                    "default_value": 50,
+                    "default_value": 90,
                     "minimum_value": "1",
-                    "minimum_value_warning": "min_bead_width / line_width * 100",
```

Comment:
I don't think this minimum warning should be removed.

Reply:
I behaved strangly (did not calculate the correct value). I think it had to do with the min_bead_width value calculation which uses this setting (recursive?). That is why I removed it. Is there another way to do this?

Reply:
You could just set it to 25%...

Reply:
I will undo the change

Reply:
> You could just set it to 25%...

That is as random as any other number. The maximum warning level was also set at 80% in the past I believe and we past that number with the current default. So might as well not set a warning level at all in my opinion.